stripper/ze_shroomforest2_p6.cfg

```diff
@@ -715,4 +715,206 @@ filter:
+{
+	match:
+	{
+		"targetname" "Item_Shroom_Text"
+		"classname" "env_message"
+	}
+	replace:
+	{
+		"message" "Weapon: Shroom - Disorient humans!\nDuration: 20 secs - Cooldown: 75 secs"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "GameText_Outro"
+		"classname" "env_message"
+	}
+	replace:
+	{
+		"message" "\n    CONGRATULATIONS!"
+	}
+	delete:
+	{
+		"OnUser1" "!selfAddOutputmessage <font color='#38afea'><font size='28'><b>    CONGRATULATIONS!0-1"
+		"OnUser2" "!selfAddOutputmessage <font color='#38afea'><font size='28'><b>                YOU WIN!0-1"
+	}
+	insert:
+	{
+		"OnUser1" "!selfAddOutputmessage \n    CONGRATULATIONS!0-1"
+		"OnUser2" "!selfAddOutputmessage \n                YOU WIN!0-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "Level_Case"
+		"classname" "logic_case"
+	}
+	delete:
+	{
+		"OnCase01" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[WARMUP]</u><br><font size='20'><font color='#ffffff'>      Wait for other players to join!01"
+		"OnCase02" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[LEVEL 1]</u><br><font size='20'><font color='#ffffff'>                   Find the Groke!01"
+		"OnCase03" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[LEVEL 2]</u><br><font size='20'><font color='#ffffff'>                   Destroy the TV!01"
+		"OnCase04" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[LEVEL 3]</u><br><font size='20'><font color='#ffffff'>              Explore the heaven!01"
+		"OnCase05" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[EXTREME 1]</u><br><font size='20'><font color='#ffffff'>       Face off the Groke once more!01"
+		"OnCase06" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[EXTREME 2]</u><br><font size='20'><font color='#ffffff'>                   Kill the TV monster!01"
+		"OnCase07" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[EXTREME 3]</u><br><font size='20'><font color='#ffffff'>                      Save the angels!01"
+		"OnCase08" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[THE END]</u><br><font size='20'><font color='#ffffff'>   Eliminate Satan once and for all!01"
+		"OnCase09" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[BARRICADE]</u><br><font size='20'><font color='#ffffff'>         Save your ass from zombies!01"
+		"OnDefault" "GameText_LevelAddOutputmessage <font color='#38afea'><font size='27'>                <u>[BUG]</u><br><font size='20'><font color='#ffffff'>Map level didn't load correctly...01"
+	}
+	insert:
+	{
+		"OnCase01" "GameText_LevelAddOutputmessage                 [WARMUP]\n      Wait for other players to join!01"
```

Comment:
\n only works in vscript .nut files. This issue applies to pretty much everything for this file.

![](https://i.imgur.com/sLtduIc.png)